apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -424,6 +453,14 @@ export default class TutorialExplorer extends React.Component {
           return false;
         }
 
+        if (
+          searchTerm !== '' &&
+          (!tutorial.name.toLowerCase().includes(searchTerm) ||
```

Comment:
(minor) Consider lower-casing searchTerm here instead of where it's input (because otherwise there's an implicit contract there that searchTerm is already lower-cased which might accidentally get broken at some point).

Reply:
It looks like `includes` is already case-insensitive so we might not need to change any cases.

Also worth noting that we carefully unit test this core filtering functionality, and should add search to those tests.

Reply:
hmmm I think `includes` is case-sensitive based on playing around with it in the console.

Reply:
You're right.  My bad, I misread some documentation.